src/Compilers/Core/Portable/DiagnosticAnalyzer/AnalyzerDriver.cs

```diff
@@ -26,11 +25,10 @@ internal abstract class AnalyzerDriver : IDisposable
         private const int MaxSymbolKind = 100;
 
         private readonly ImmutableArray<DiagnosticAnalyzer> _analyzers;
-        private readonly CancellationTokenRegistration _queueRegistration;
         protected readonly AnalyzerManager analyzerManager;
 
-        // Lazy fields initialized in Initialize() API
-        private Compilation _compilation;
+        // Lazy fields initialized in Initialize() or StartPrimaryTask() API
```

Comment:
is driver re-used. why some of field is not readonly when it is lazily initialized field?